### DISCUSSION TITLE
How to use "Favorites"?

### DISCUSSION THREAD

**Discussion by df8oe** [magicbug/Cloudlog#2884] [Q&A] [answered]
Hi,
"QSO -> Live QSO" I see "Favorites dropdown". I can add a favorite - and if I log next QSO I can delete it at the same dropdown. But how to quick access a favorite for a QSO? I haven't found any possibility to do that...

**Comment by HB9HIL** [comment, marked as answer]
Don't click on the trash sign on the Dropdown. Just click on the Favourite itself 😉

**Comment by df8oe** [comment]
Tnx! Quite simple - but not intuitive. Possible improvement also quite simple: Add something like | or color "red" to the trashcan that show: the line is not one action - it is two actions...

**Reply by HB9HIL** [reply]
Yep already working on this :)

**Reply by HB9HIL** [reply]
@df8oe better?

<img width="432" alt="fav_dropdown" src="https://github.com/magicbug/Cloudlog/assets/80885850/6e8717ac-9b08-4f53-b632-2997305b8ffb">

**Comment by df8oe** [comment]
Muuuuuch better!!

**Comment by df8oe** [comment]
Additional info: I was confused because hovering the mouse pointer over the line selected the whole line (trashcan AND favorite) - so I thought it is one action. Possibly you can change this behaviour with a css entry? Hovering makes trashcan OR mode active?

**Reply by HB9HIL** [reply]
I think with the new design it's not needed at the moment. 

The new style is merged in `dev` branch. Will be published in the next release. Probably next year 😆

**Comment by df8oe** [comment]
Thank you for all your work. Next year sounds "far away" - but it is only 4 days.... I am running experimental/dev branch because I want to contribute by creating issue reports... Stay healthy and have a good start to 2024!

**Reply by HB9HIL** [reply]
THATS GREAT, without feedback we can't make Cloudlog better. 

For you the same 

"Rutsch gut rüber" 😄
